src/style.css

```diff
@@ -0,0 +1,128 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  font-size: 14px;
+  box-sizing: border-box;
+}
+
```

Comment:
add a box-sizing: border-box property for body and all elements, your input now is broader than it should